docs/data-sources/federated_settings_identity_providers.md

```diff
@@ -2,6 +2,8 @@
 
 `mongodbatlas_federated_settings_identity_providers` provides an Federated Settings Identity Providers datasource. Atlas Cloud Federated Settings Identity Providers provides federated settings outputs for the configured Identity Providers.
 
+Note: The implementation is currently limited to returning a maximum of 100 results.
```

Comment:
Suggest using positive statement by presenting it as the intended behavior rather than a limitation:

Note: This implementation returns a maximum of 100 results.